src/apis/rankingApi.mock.ts

```diff
@@ -0,0 +1,18 @@
+import { http, HttpResponse } from 'msw';
+import { mockRanking } from '@/apis/mockData';
+
+export const rankingHandlers = [
+  http.get('/api/products/ranking', ({ request }) => {
+    const url = new URL(request.url);
+    const targetType = url.searchParams.get('targetType') || 'ALL';
+    const rankType = url.searchParams.get('rankType') || 'MANY_WISH';
+
+    const filteredData = mockRanking.filter(
+      p => p.gender === targetType && p.action === rankType
```

Comment:
p가 뭔지 알기 어려우니까 조금 이름을 풀어서 써주면 좋을 거 같고요!

```suggestion
      p => p.gender === targetType && p.action === rankType => isSameFilter
```
와 같은 식으로 별도의 함수로 분리하면 더 가독성이 올라갈 거 같습니다!